### ISSUE TITLE
AArch64: Implement fast byte reversal Java methods

### ISSUE THREAD

**Issue by 0xdaryl** [eclipse/openj9#7127]
Map the following recognized methods to byte reversal instructions (the `REV` family) on AArch64:
```
TR::java_lang_Long_reverseBytes
TR::java_lang_Integer_reverseBytes
TR::java_lang_Short_reverseBytes
```

**Comment by knn-k** [comment]
https://github.com/eclipse/omr/pull/5563 implemented byteswap evaluators for the `reverseBytes()` methods.

**Comment by 0xdaryl** [comment]
Implemented in #5563.  Closing.
